rules-reviewed/jboss-eap/java-ee/java-ee-jaxrpc.windup.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<ruleset id="java-ee-jaxrpc" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            This ruleset provides generic migration knowledge for the Java EE specification
```

Comment:
I wasn't really sure. This is something we want included when the EAP ruleset is run, and that's what we're focusing on for now, so I put it here. That was my reasoning anyway.